deployment/Dockerfile.fedora.amd64

```diff
@@ -1,4 +1,4 @@
-FROM fedora:33
+FROM fedora:36
```

Comment:
Okay, good thing I tried playing through the workflow of our current CI.
It seems that `fedora:36` does not bundle `make`, nor will it get installed alongside the packages that we install below (odd that automake does not bundle it).
This obviously leads to a build error once we attempt to run the Makefile to build the fedora RPM. 
Regardless, if you add `make` to the `dnf install` call below it builds as you would expect.

Could you please add `make` to the package list below, so that it gets installed and won't error the CI, after that I consider this PR done from my point of view.

Reply:
Woops, missed this review earlier.

Added and pushed